src/main/java/hudson/plugins/git/UserMergeOptions.java

```diff
@@ -39,7 +40,6 @@ public UserMergeOptions(String mergeRemote, String mergeTarget, String mergeStra
      * @param mergeStrategy merge strategy
      * @param fastForwardMode fast forward mode
      */
-    @DataBoundConstructor
     public UserMergeOptions(String mergeRemote, String mergeTarget, String mergeStrategy,
```

Comment:
Shouldn't this constructor be marked as `@Deprecated` as the one in line 33? Then both could be removed in v4.0.0.

Reply:
It _could_ be deprecated. I would not recommend deleting it so quickly anyway—leave it there for a year or two.